cypress/e2e/nestedDialogs.cy.js

```diff
@@ -13,15 +13,15 @@ describe('Nested dialogs', () => {
   })
 
   it('should close only the top most dialog when pressing ESC', () => {
-    cy.get('body').trigger('keydown', { keyCode: 27, which: 27 })
+    cy.get('html').trigger('keydown', { key: 'Escape', keyCode: 27, which: 27 })
```

Comment:
First , the `keydown` event is bound to the document, and not the `body` element (although I’m pretty sure it doesn’t matter in that case). Secondly, the event listener looks at the `key` property of the event since [this commit](https://github.com/KittyGiraudel/a11y-dialog/commit/901bac3175b2f8ea41efc45a6f269d9872eea1f0), so this was just not working.